sway-lib-std/src/math.sw

```diff
@@ -6,6 +6,24 @@ pub trait Root {
     fn sqrt(self) -> Self;
 }
 
+impl Root for u256 {
+    /// Integer square root using [Newton's Method](https://en.wikipedia.org/wiki/Integer_square_root#Algorithm_using_Newton's_method).
```

Comment:
Trait implementation functions are supposed to show the documentation written where the trait functions are defined, not a separate definition for every implementation, as such please move this into the function as a comment (2 slashes instead of 3)